homeassistant/components/fido/sensor.py

```diff
@@ -84,45 +194,31 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     sensors = []
     for number in fido_data.client.get_phone_numbers():
         for variable in config[CONF_MONITORED_VARIABLES]:
-            sensors.append(FidoSensor(fido_data, variable, name, number))
+            sensors.append(
+                FidoSensor(fido_data, variable, name, number, SENSOR_TYPES[variable])
+            )
```

Comment:
This needs to be refactored then as well. Something like
```py
for description in SENSOR_TYPES:
    if description.key in config[CONF_MONITORED_VARIABLES]:
        sensors.append(
            FidoSensor(fido_data, description, name, number)
        )
```

Don't pass `variable` or `SENSOR_TYPES[variable]` to `FidoSensor`, just `description`.